[WIP]: IBMZ updates for HCP 4.17 (#83206)
openshift/openshift-docs: modules/hcp-ibmz-lpar-agents.adoc

<3> For installations on DASD-type disks, use `rd.dasd` to specify the DASD where {op-system-first} is to be installed. For installations on FCP-type disks, use `rd.zfcp=<adapter>,<wwpn>,<lun>` to specify the FCP disk where {op-system} is to be installed.
<4> Specify this parameter when you use an Open Systems Adapter (OSA) or HiperSockets.

> . Download the the `.ins` and `initrd.img.addrsize` files from InfraEnv.

Reviewer: ```suggestion
. Download the the `.ins` and `initrd.img.addrsize` files from the `InfraEnv` resource.
```

+

> By default, the URL for .ins` and `initrd.img.addrsize` is not available in the InfraEnv. You must edit th existing URL to fetch these artifacts.

Reviewer: ```suggestion
By default, the URL for the `.ins` and `initrd.img.addrsize` files is not available in the `InfraEnv` resource. You must edit the URL to fetch those artifacts.
```

+

> Update the kernel URL endpoint to include ins-file as given below:

Reviewer: ```suggestion
..  Update the kernel URL endpoint to include `ins-file`:
```

+
.Example URL
[source,yaml]
----
https://…/boot-artifacts/ins-file?arch=s390x&version=4.17.0
----
+

> Update the `initrd` URL endpoint to include `s390x-initrd-addrsize` as given below:

Reviewer: ```suggestion
.. Update the `initrd` URL endpoint to include `s390x-initrd-addrsize`:
```

+
.Example URL
[source,yaml]
----
https://…./s390x-initrd-addrsize?api_key=<api-key>&arch=s390x&version=4.17.0
----

> . Transfer the `initrd`, `kernel`, `generic.ins`, and `initrd.img.addrsize` parameter files to the file server. For more information about how to transfer the files with FTP and boot, see _Installing in an LPAR_.

Reviewer: ```suggestion
. Transfer the `initrd`, `kernel`, `generic.ins`, and `initrd.img.addrsize` parameter files to the file server. For more information about how to transfer the files with FTP and boot, see "Installing in an LPAR".
```

<3> For installations on DASD-type disks, use `rd.dasd` to specify the DASD where {op-system-first} is to be installed. For installations on FCP-type disks, use `rd.zfcp=<adapter>,<wwpn>,<lun>` to specify the FCP disk where {op-system} is to be installed.
<4> Specify this parameter when you use an Open Systems Adapter (OSA) or HiperSockets.

. Download the the `.ins` and `initrd.img.addrsize` files from the `InfraEnv` resource.
+

> By default, the URL for the `.ins` and `initrd.img.addrsize` files is not available in the `InfraEnv` resource. You must edit the URL to fetch those artifacts...

Reviewer: ```suggestion
By default, the URL for the `.ins` and `initrd.img.addrsize` files is not available in the `InfraEnv` resource. You must edit the URL to fetch those artifacts.
```

+

> .. Update the kernel URL endpoint to include `ins-file`:

Reviewer 2: In this update you've removed commands and in place have direct steps with no commands but examples. Would it be better to offer the user commands for how to update the kernel URL endpoint etc? It could be that I'm not familiar with this so my these may be obvious commands.

> ..  Update the kernel URL endpoint to include `ins-file`:

Reviewer: ```suggestion
.. Update the kernel URL endpoint to include `ins-file`:
```
